fusor-ember-cli/app/utils/foreman-task-util.js

```diff
@@ -0,0 +1,30 @@
+// ForemanTaskUtil: Wrappers around driving foreman tasks
+
+import Ember from 'ember';
+import request from 'ic-ajax';
+let Promise = Ember.RSVP.Promise;
+
+class ForemanTaskUtil {
+  constructor(csrfToken) {
+    this._csrfToken = csrfToken;
+    this._uriRoot = `${window.location.protocol}//${window.location.host}`;
+    this._foremanApiPath = '/foreman_tasks/api';
+    this._foremanApiUri = `${this._uriRoot}${this._foremanApiPath}`;
+    this._resumePath = `/tasks/bulk_resume`;
+    this._resumeUri = `${this._foremanApiUri}${this._resumePath}`;
```

Comment:
It shouldn't change since `v2` is the default, so the all API URL's can be written between /v2/.  It's optional.